src/Raven.Server/Documents/Handlers/Debugging/AdminDebugInfoPackageHandler.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Server.Documents.Handlers.Admin;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.DebugInfo;
+using Raven.Server.Web;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Documents.Handlers.Debugging
+{
+    public class AdminDebugInfoPackageHandler : AdminRequestHandler
+    {
+        private static readonly IReadOnlyList<RouteInformation> _debugInformationRoutes = 
+            RouteScanner.Scan(attr => attr.IsDebugInformationEndpoint && 
+                                      attr.Path.Contains("info-package") == false).Values.ToList();
+
+        [RavenAction("/debug/info-package", "GET", IsDebugInformationEndpoint = true)]
+        public async Task GetInfoPackage()
+        {            
+            var contentDisposition = $"attachment; filename={DateTime.UtcNow}.debug-info-package.zip";
+            HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            {
+                using (var ms = new MemoryStream())
```

Comment:
ZipArchive tries to use not supported property of Kestrel's response stream -> if I try to write directly to response stream I get Kestrel exception